projects/HTAI

```diff
@@ -0,0 +1,10 @@
+const { getUniTVL } = require('../helper/unknownTokens')
+
+module.exports = {
+  misrepresentedTokens: true,
+  methodology:
+    "Factory address (0x6402613eE4167832a29531ca67170c91a3304A1d) is used to find the LP pairs. TVL is equal to the liquidity on the AMM.",
+  theta: {
+    tvl: getUniTVL({ factory: '0x6402613eE4167832a29531ca67170c91a3304A1d', useDefaultCoreAssets: true }),
```

Comment:
this is LP contract address? and why are you checking theta chain

Reply:
why are you trying to count the LP value as your tvl, did your team deploy the dex?